test/integration/connect/conftest.py

```diff
@@ -0,0 +1,96 @@
+import subprocess
+import pytest
+import os
+
+from project import Project
+from project import patch_cartridge_proc_titile
+from project import remove_dependency
+from project import replace_project_file
+from project import INIT_NO_CARTRIDGE_FILEPATH
+
+from utils import DEFAULT_CFG, DEFAULT_RPL_CFG
+from utils import ProjectWithTopology, Instance
```

Comment:
why imports from ``project`` looks different from ``utils``? import from utils seems to me (and pep8) better